network/marks.csv

```diff
@@ -116,6 +116,7 @@ OHIN,CG,no,Ohingaiti,-39.918289832,175.790675494,530.658,NZGD2000,2008-09-03T00:
 OKOH,CG,no,Okoha,-41.019262,174.060283,334,NZGD2000,2009-11-16T00:00:00Z,9999-01-01T00:00:00Z
 OPTK,CG,no,Opotiki,-38.046521999,177.307578963,127.477,NZGD2000,2009-08-20T00:00:00Z,9999-01-01T00:00:00Z
 OROA,CG,no,Oueroa,-40.10435506,176.68067166,279,WGS84,2010-11-08T00:00:00Z,9999-01-01T00:00:00Z
+ORRG,CG,no,Mangere,36.9726410056,174.770495275,47.245,WGS84,2024-12-05T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
Hey @EvanCGNS ! I believe we are still in the southern hemisphere, so the latitude has to be negative.

Reply:
Thanks Aleks! Good catch